e2e/images/metrics-api/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.17
+
+
+WORKDIR /app
+COPY go.mod .
+COPY server.go server.go
+
+RUN go mod tidy
+RUN go build -o server .
+
+EXPOSE 8080
+
+CMD ["/app/server"]
```

Comment:
Please, use multistage build. [Postgresql image](https://github.com/kedacore/test-tools/blob/main/e2e/images/postgresql/Dockerfile) can be an example